src/main/kotlin/blackjack/domain/game/BlackJackGameTurn.kt

```diff
@@ -0,0 +1,16 @@
+package blackjack.domain.game
+
+import blackjack.domain.player.PlayerName
+
+sealed class BlackJackGameTurn {
+
+    object CardDistributionWait : BlackJackGameTurn()
+
+    data class HitAnswerWait(
+        val playerName: PlayerName,
+    ) : BlackJackGameTurn()
```

Comment:
```suggestion
    object CardDistribution : BlackJackGameTurn()

    data class Hit(
        val playerName: PlayerName,
    ) : BlackJackGameTurn()
```
블랙잭이라는 도메인을 이해하고 있다면, Wait은 없어도 될 것 같아요. 😅

Reply:
다소 명확하게 표현하고자 함이었는데요, 말씀 주신 것 처럼 도메인이 그 의미가 충분히 포함되어 있는 것 같네요. 반영했습니다 :)
https://github.com/next-step/kotlin-blackjack/pull/465/commits/e51c23aae4323f4166774325e68f87960849fa21